nxdrive/options.py

```diff
@@ -149,7 +149,7 @@ def __call__(self, new_value: bool, /) -> bool:
 class SynchronzationHandler(CallableFeatureHandler):
     def __call__(self, new_value: bool, /) -> bool:
         option_updated = super().__call__(new_value)
-        if option_updated:
+        if option_updated and Options.synchronization_enabled != new_value:
```

Comment:
The parent's method will tell us if the option was updated, it seems a redundant check, no?